server/service/src/invoice_line/stock_in_line/update/generate.rs

```diff
@@ -140,6 +141,7 @@ fn generate_line(
         currency_id,
         currency_rate,
     )?;
+    update_line.item_variant_id = item_variant_id.or(update_line.item_variant_id);
```

Comment:
This pattern would mean we can't update it to null once it's been set right?
Not a big deal for this PR but feels like something we'd want to be able to do right?

Reply:
That's a great point, it probably should be nullable update - can fix with the UI PR!